LICENSE.rst

```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2023, Scientific Python
```

Comment:
I wonder whether it should be `Scientific Python Developers`? That would better align with the libraries (numpy and scipy both  have the license to the devs, mpl to the dev team) rather than to the corporates where it's just the name of the org. Currently it's a bit of a mixture in SP, but we're still in early days to adjust if we want consistency.